fe1-web/src/features/wallet/objects/__tests__/RollCallToken.test.ts

```diff
@@ -0,0 +1,23 @@
+import { mockKeyPair, mockLao, mockRC } from '__tests__/utils';
+import { PopToken } from 'core/objects';
+
+import { RollCallToken } from '../RollCallToken';
+
+const mockToken = PopToken.fromState(mockKeyPair.toState());
+
+describe('Roll call token object', () => {
+  it('throws when token is undefined', () => {
+    expect(() => new RollCallToken({ token: undefined })).toThrow(Error);
+  });
+  it('throws when laoId is undefined', () => {
+    expect(() => new RollCallToken({ token: mockToken })).toThrow(Error);
+  });
+  it('throws when rollCallId is undefined', () => {
+    expect(() => new RollCallToken({ token: mockToken, laoId: mockLao.id })).toThrow(Error);
+  });
+  it('throws when rollCallName is undefined', () => {
+    expect(
+      () => new RollCallToken({ token: mockToken, laoId: mockLao.id, rollCallId: mockRC.id }),
+    ).toThrow(Error);
+  });
```

Comment:
Nice! Maybe just add one more test where you call the constructor with all required arguments and check `expect().not.toThrow(Error)` and optionally whether the fields of the return object are set to the expected values?

Reply:
Yes !